packages/transactions/src/builders.ts

```diff
@@ -1692,3 +1663,30 @@ export async function estimateTransactionFeeWithFallback(
     throw error;
   }
 }
+
+/** @internal multi-sig signing re-use */
+function mutatingSignMultiSig(
```

Comment:
This function can sign *or* append a pubkey. Seems like the name should reflect that